oidc-api/src/main/java/uk/gov/di/authentication/oidc/entity/AuthUserInfoClaims.java

```diff
@@ -12,6 +12,7 @@ public enum AuthUserInfoClaims {
     EMAIL_VERIFIED("email_verified"),
     PHONE_NUMBER("phone_number"),
     PHONE_VERIFIED("phone_number_verified"),
+    WALLET_SUBJECT_ID("https://vocab.account.gov.uk/v1/walletSubjectID"),
```

Comment:
Really picky but it might be nice to keep all the id's together

Reply:
This is not an auth user info claim